packages/flutter_tools/lib/src/ios/xcode_debug.dart

```diff
@@ -411,6 +413,49 @@ class XcodeDebug {
       verboseLogging: verboseLogging,
     );
   }
+
+  /// Ensure the Xcode project is set up to launch an LLDB debugger. If these
```

Comment:
does this check only need to be done when we're initiating debugging via the Xcode GUI?

Reply:
Yes. Also, only need to be done for `BuildableIOSApp` (so not prebuilt) since prebuilt uses a Xcode project we generate on the fly.